tests/maps/test_map_validators.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+import pytest
+from django.conf import settings
+from django.contrib.gis.geos import GEOSGeometry
+from django.forms import ValidationError
+
+from adhocracy4.maps.validators import PointInPolygonValidator
+
+
+@pytest.mark.django_db
+def test_point_in_polygon_validator_valid_point():
+    polygon = GEOSGeometry(
+        json.dumps(settings.BERLIN_POLYGON["features"][0]["geometry"])
+    )
+    validator = PointInPolygonValidator(polygon=polygon)
+
+    # point within the berlin polygon
+    geojson_point = {
+        "type": "Feature",
+        "geometry": {
+            "type": "Point",
+            "coordinates": [13.411924777644563, 52.499598134440944],
+        },
+    }
+    point = GEOSGeometry(json.dumps(geojson_point["geometry"]))
```

Comment:
same, this could be named either `gis_point` or `geos_point`, just stay consistent.